datadog/dogstatsd/base.py

```diff
@@ -47,8 +53,8 @@
 class DogStatsd(object):
     OK, WARNING, CRITICAL, UNKNOWN = (0, 1, 2, 3)
 
-    def __init__(self, host=DEFAULT_HOST, port=DEFAULT_PORT, max_buffer_size=50, namespace=None,
-                 constant_tags=None, use_ms=False, use_default_route=False,
+    def __init__(self, host=DEFAULT_HOST, port=DEFAULT_PORT, max_buffer_size=MAX_BUFFER_ITEM, max_buffer_bytes_size=0,
+                 namespace=None, constant_tags=None, use_ms=False, use_default_route=False,
```

Comment:
We should add new parameter at the end of the parameter list, so we don't break customer code who pass arguments without naming them.

ie:
```python
DogStatsd("localhost", 1234, 40, ["const_tag1", "const_tag2"])
```

Reply:
It makes sense.